buildenv/jenkins/common/build.groovy

```diff
@@ -404,6 +416,12 @@ def archive_sdk() {
                 env.CUSTOMIZED_SDK_URL = "${ARTIFACTORY_CONFIG[ARTIFACTORY_CONFIG['defaultGeo']]['url']}/${ARTIFACTORY_CONFIG['uploadDir']}${SDK_FILENAME}"
                 currentBuild.description += "<br><a href=${CUSTOMIZED_SDK_URL}>${SDK_FILENAME}</a>"
 
+                if (!zelixFiles.isEmpty()) {
+                    for (String zelixFile in zelixFiles) {
+                        ZELIX_URL = "${ARTIFACTORY_CONFIG[ARTIFACTORY_CONFIG['defaultGeo']]['url']}/${ARTIFACTORY_CONFIG['uploadDir']}${zelixFile}"
```

Comment:
After the suggested change above, we may want to trim the prefix `${makeDir}/` from each `zelixFile`.